flycheck.el

```diff
@@ -9276,6 +9277,28 @@ See URL `http://www.veripool.org/wiki/verilator'."
           line ": " (message) line-end))
   :modes verilog-mode)
 
+(flycheck-def-option-var flycheck-ghdl-language-standard nil vhdl-ghdl
+  "The language standard to use in GHDL.
+
+The value of this variable is either a string denoting a language
+standard, or nil, to use the default standard.  When non-nil,
+pass the language standard via the `--std' option."
+  :type '(choice (const :tag "Default standard" nil)
+                 (string :tag "Language standard"))
+  :safe #'stringp
+  :package-version '(flycheck . "0.31"))
+(make-variable-buffer-local 'flycheck-ghdl-language-standard)
+
+(flycheck-define-checker vhdl-ghdl
+  "A VHDL syntax checker using GHDL."
+  :command ("ghdl"
+            "-s"
```

Comment:
What's `-s` for?  Could you add comment to the source that explains the purpose of the flag?